asapdiscovery-workflows/asapdiscovery/workflows/docking_workflows/cross_docking.py

```diff
@@ -204,8 +206,12 @@ def cross_docking_workflow(inputs: CrossDockingWorkflowInputs):
         use_dask=inputs.use_dask,
         dask_client=dask_client,
         failure_mode=inputs.failure_mode,
+        return_for_disk_backend=True,
     )
 
+    if isinstance(results[0], list):
```

Comment:
Lets sort this out inside the docking function, will be easier with seperate branches for single and multistructure logic

Reply:
@apayne97 had the right idea and I was confused. No longer relevant.